apps/frontend/src/renderer/components/github-issues/components/AutoPRReviewProgressCard.tsx

```
@@ -0,0 +1,579 @@
+  t: (key: string) => string;
+}) {
+  const { total, passed, failed, pending } = summary;
+
+  if (total === 0) {
+    return (
+      <span className="text-sm text-gray-500">{t('autoPRReview.noChecks')}</span>
+    );
+  }
+
+  return (
+    <div className="flex items-center gap-2 text-sm" aria-label={`CI: ${passed}/${total} passed`}>
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)